clinica/utils/stream.py

```diff
@@ -40,5 +40,5 @@ def cprint(msg):
     if clinica_verbose is True:
         print(msg)
     else:
-        print("@clinica@ %s" % msg)
+        print("@clinica@%s\n" % msg)
```

Comment:
Pourquoi vouloir coller '@clinica@' avec le texte ? C'est beaucoup plus lisible `@clinica@ Mon message` que `@clinica@Mon message. Pareil pour le saut de ligne, je l'avais enlevé parceque ca rendait vraiment disgracieux l'affichage.

Reply:
L'idée originale du `cprint` était d'afficher dans le terminal uniquement nos messages envoyés par `cprint` et de cacher tous les autres messages (ceux de Nipype, `print`, etc.) quand le mode 'verbose' était désactivé .

Donc, quand tu faisais un :
```
my_string="My message to display"
```
cela affichait un :
```
My message to display
```
au lieu d'un :
```
@linica@ My message to display
```
que l'on a depuis quelques mois.

Et lorsque tu utilises l'option 'verbose', les `cprint` et autres messages de Nipype & cie cohabitent sans que la chaîne de caractère `@clinica@` ne soit affichée quelque part :
![CPrintComparison](https://user-images.githubusercontent.com/24808663/63517553-77581580-c4ef-11e9-99fb-2009624ec380.png)


Cependant lorsque les lignes suivantes avait été rajoutées :
```
sys.stdout = open(os.devnull, 'w')
from nipype import config
sys.stdout = sys.__stdout__
```
pour cacher le message "Assuming non interactive session since isatty found missing", cela a cassé ce comportement. (cf [Issue #15](https://github.com/aramis-lab/clinica/issues/7) ). C'est pour cela que j'ai fait le commit de régression cb66c79 puis le bugfix alternative dans le commit f87e451.